packages/input_schema/src/utilities.ts

```diff
@@ -69,7 +69,10 @@ function validateProxyField(
     // If options are not provided skip additional checks
     if (!options) return fieldErrors;
 
-    const selectedProxyGroups = (apifyProxyGroups || []);
+    let selectedProxyGroups = [];
+    if (apifyProxyGroups) {
+        selectedProxyGroups = Array.isArray(apifyProxyGroups) ? apifyProxyGroups : [apifyProxyGroups];
```

Comment:
I think it would be better to be strict here, we should check that if `apifyProxyGroups` is defined, it is an array of strings, and throw an error otherwise. This would still let through numbers, objects and other non-string things, which would then create a mess later.